src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisFilters.tsx

```diff
@@ -53,14 +52,13 @@ export interface AutomatedAnalysisFiltersProps {
 
 export const AutomatedAnalysisFilters: React.FC<AutomatedAnalysisFiltersProps> = ({ updateFilters, ...props }) => {
   const dispatch = useDispatch<StateDispatch>();
-  const { t } = useTranslation();
 
   const currentCategory = useSelector((state: RootState) => {
-    const targetAutomatedAnalysisFilters = state.automatedAnalysisFilters.state.targetFilters.filter(
+    const targetAutomatedAnalysisFilters = state.automatedAnalysisFilters.targetFilters.filter(
       (targetFilter) => targetFilter.target === props.target
     );
     if (!targetAutomatedAnalysisFilters.length) {
-      return t('FILTER_NAME', { ns: 'common' });
+      return 'Name';
```

Comment:
Why does this need to be changed this way? Is this not actually a display string but an internal lookup key?

Reply:
Right exactly: https://github.com/cryostatio/cryostat-web/blob/8b8de35adfb0951efc7c3de1b7f6883c2e940044/src/app/Shared/Redux/Filters/AutomatedAnalysisFilterSlice.tsx#L199

Reply:
Previously, using locale will cause the filters to be lost every time language is changed. Tho, we only have `en` for the moment.

Reply:
But seems like its also used as display. I will fix that.

Reply:
Fixed now :)) Thanks!